mcs/class/corlib/corlib_xtest.dll.sources

```diff
@@ -146,6 +146,7 @@
 
 # System.Runtime
 # TODO: Many more to be included
+../../../external/corefx/src/System.Runtime/tests/System/HashCodeTests.netcoreapp.cs
```

Comment:
Please also include

`../../../external/corefx/src/System.Security.Cryptography.Algorithms/tests/RandomNumberGeneratorTests*.cs` as we are switching the implementation